### DISCUSSION TITLE
Add "details pan on the bottom" & "hide folders from this PC" options like OldNewExplorer

### DISCUSSION THREAD

**Discussion by Ben2557** [valinet/ExplorerPatcher#1130] [Ideas]
I'm using a windows 7-like Theme in Windows 10.

I really love your ExplorerPatcher, I think you could add "details pan on the bottom" & "hide folders from this PC" as OldNewExplorer do.
I need to combine using both software to enable it.
![image](https://user-images.githubusercontent.com/91072875/226173053-2ecc6d4d-5ffb-4735-8f1b-e3dde34290d5.png)
![image](https://user-images.githubusercontent.com/91072875/226173092-76d1eaac-f32e-48bb-977c-5f39eb1eeb3e.png)

My global File explorer :
![image](https://user-images.githubusercontent.com/91072875/226173181-bea97092-6f2c-441e-b101-ee05d167f837.png)

OldNewExplorer :
![image](https://user-images.githubusercontent.com/91072875/226173204-6e87a085-2f67-4bac-90d9-4eb37e0844d5.png)

**Comment by MANTISInf** [comment]
I like this too.

**Comment by pyrates999** [comment]
Please close this as a duplicate of https://github.com/valinet/ExplorerPatcher/discussions/1282

**Comment by ourslfs** [comment]
btw is that windowblinds?

**Reply by Ben2557** [reply]
Yes it is
